src/library/dao/cue.h

```diff
@@ -21,6 +21,8 @@ class Cue : public QObject {
         BEAT    = 3,
         LOOP    = 4,
         JUMP    = 5,
+        BEGIN   = 6,
```

Comment:
I think it might make more sense to call these "fade in" and "fade out" cues (or autodj_in / autodj_out), since that's more aligned with the purpose.

Tracks may not have a beginning or end, and user's probably don't particularly care about having the beginning and end of the track annotated, but they do care about annotating where AutoDJ would fade in and out. Plus if they edit the cue to be located where they want AutoDJ to fade in, then it doesn't point at the track start anymore so it'd be misnamed.